environments/icds-cas/app-processes.yml

```diff
@@ -218,7 +218,7 @@ celery_processes:
       pooling: gevent
       concurrency: 10
       num_workers: 2
-  celery35,celery36,celery37,celery38,celery39,celery40,celery41,celery42,celery43,celery44,celery45,celery46,celery47:
+  celery0,celery1,celery2,celery3,celery4,celery5,celery6,celery7,celery8,celery9,celery10,celery11,celery12:
```

Comment:
aren't these old machines?

Reply:
This is for removing existing running queue on those machines.